knockout_modeler/tests.py

```diff
@@ -84,7 +86,7 @@ def setup_user(self):
         wayne = Person()
         wayne.first_name = 'Lil'
         wayne.last_name = 'Wayne'
-        wayne.uuid = 'dead-beef'
+        wayne.uuid = uuid.uuid4()
```

Comment:
Since `setup_user` is called multiple times in a single test case, the `uuid` of this object needed to be unique.